students/rachel_s/session03/list_lab.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+og_fruits = fruits.copy()
```

Comment:
Minor: Wow, I never knew this and learned this from you on this occasion! I think most people would still recommend you to use `fruits[:]` instead of `fruits.copy()`, though. Just a little minor thing...